README.md

```diff
@@ -1 +1,161 @@
-# audit-service-deploy
\ No newline at end of file
+## Audit Service Deployment
+
+### Introduction
+
+The Audit Service is designed to provide auditing of user and system actions by defining the required events and the information associated with each event.  This definition is registered with the Audit service and used to create a Java SDK to record the audited events.  The Audit service is multi-tenant aware.  Applications send events using the generated Java SDK to Elasticsearch where they are indexed according to each tenant.
```

Comment:
"This definition is registered with the Audit service": This is not true.
"and used to create a Java SDK": It can be used for this but doesn't necessarily have to be.